app/src/test/java/com/example/se2_exploding_kittens/FavorCardTest.java

```diff
@@ -0,0 +1,43 @@
+package com.example.se2_exploding_kittens;
+
+import static com.example.se2_exploding_kittens.game_logic.cards.DefuseCard.DEFUSE_CARD_ID;
+import static com.example.se2_exploding_kittens.game_logic.cards.FavorCard.FAVOR_CARD_ID;
+import static org.mockito.Mockito.mock;
+
+import com.example.se2_exploding_kittens.game_logic.Deck;
+import com.example.se2_exploding_kittens.game_logic.DiscardPile;
+import com.example.se2_exploding_kittens.game_logic.Player;
+import com.example.se2_exploding_kittens.game_logic.cards.DefuseCard;
+import com.example.se2_exploding_kittens.game_logic.cards.FavorCard;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FavorCardTest {
```

Comment:
These should remain public since the actions within cards do not always have the same signature, there may be need to call them from another package like the deck Class which is not in the card package.